chart/values.yaml

```diff
@@ -650,6 +650,7 @@ workers:
     args: ["bash", "/clean-logs"]
     # Number of days to retain logs
     retentionDays: 15
+    deleteLogFolders: true
```

Comment:
If we do keep it confugurable, I think `deleteFolders` is better. Like with `retentionDays`, we know we are dealing with logs already.